webapp/templates/jury/contest.html.twig

```diff
@@ -27,6 +27,11 @@
             This contest is final.
         </div>
     {% endif %}
+    {%- if is_granted('ROLE_ADMIN') and contest.isLocked -%}
+        <div class="alert alert-danger">
+            This contest is locked. Unlock it to edit contest data.
```

Comment:
As you're not actually editing yet, and this is supposed to be a 'normal operation situation', maybe `danger` is too high a class and warning or info would suffice?